tests/jwt_test.py

```diff
@@ -28,6 +28,41 @@ def test_refresh_jwt_issuer(network, args):
     return network
 
 
+@reqs.description("Multiple JWT issuers registered at once")
+def test_jwt_endpoint(network, args):
+    primary, _ = network.find_nodes()
+
+    issuers = [
+        infra.jwt_issuer.JwtIssuer("issuer1"),
+        infra.jwt_issuer.JwtIssuer("issuer2"),
+    ]
+    for issuer in issuers:
+        LOG.info(f"Add JWT issuer {issuer.name}")
+        kid = f"kid_{issuer.name}"
+        with tempfile.NamedTemporaryFile(prefix="ccf", mode="w+") as metadata_fp:
+            json.dump({"issuer": issuer.name}, metadata_fp)
+            metadata_fp.flush()
+            network.consortium.set_jwt_issuer(primary, metadata_fp.name)
+
+        with tempfile.NamedTemporaryFile(prefix="ccf", mode="w+") as jwks_fp:
+            json.dump(issuer.create_jwks(kid), jwks_fp)
+            jwks_fp.flush()
+            network.consortium.set_jwt_public_signing_keys(
+                primary, issuer.name, jwks_fp.name
+            )
+
+    with primary.client(network.consortium.get_any_active_member().local_id) as c:
+        r = c.get("/gov/jwt_keys/all")
+        assert r.status_code == 200, r
+        info = r.body.json()
+        LOG.error(info)
+        for issuer in issuers:
```

Comment:
Can we extend this to test multiple kids-per-issuer? I'm not sure why we're changing the API to return both if we're not testing that it is correctly supported + fetched.

So rather than having `issuers = [ ... ]`, we'd have `keys = { infra.jwt_issuer.JwtIssuer("issuer1"): [ "issuer1_kid1", "issuer2_kid2" ]... }`. I think that should be supported everywhere, though `create_jwks` might need to be modified to take a list of kids.

Reply:
Good point. This is now done.